packages/web-components/src/components/dotcom-shell/dotcom-shell-container.ts

```diff
@@ -0,0 +1,256 @@
+ */
+
+import { pickBy } from 'lodash-es';
+import { html, property, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import { BasicLink, BasicLinkSet, MastheadLink } from '../../globals/services-store/types/translateAPI';
+import { USER_AUTHENTICATION_STATUS } from '../../globals/services-store/types/profileAPI';
+import { FOOTER_SIZE } from '../footer/footer';
+import '../footer/footer-container';
+import { MastheadProfileItem } from '../masthead/masthead-container';
+import { LocaleModalLocaleList } from '../locale-modal/locale-modal-composite';
+import './dotcom-shell';
+import styles from './dotcom-shell-container.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Dotcom shell.
+ *
+ * @element dds-dotcom-shell-container
+ */
+@customElement(`${ddsPrefix}-dotcom-shell-container`)
+class DDSLinkWithIcon extends LitElement {
+  private _footerRenderRoot: Element | null = null;
+
+  private _mastheadRenderRoot: Element | null = null;
+
+  private _createFooterRenderRoot() {
+    const footer = this.ownerDocument.createElement(`${ddsPrefix}-footer-container`);
+    this.parentNode?.insertBefore(footer, this.nextSibling);
+    return footer;
+  }
+
+  private _createMastheadRenderRoot() {
+    const masthead = this.ownerDocument.createElement(`${ddsPrefix}-masthead-container`);
+    this.parentNode?.insertBefore(masthead, this);
+    return masthead;
+  }
+
+  /**
+   * `true` to activate the search box.
+   */
+  @property({ attribute: 'activate-search' })
+  activateSearch = false;
+
+  /**
+   * The profile items for authenticated state.
+   */
+  @property({ attribute: false })
+  authenticateProfileItems?: MastheadProfileItem[];
+
+  /**
+   * The brand name.
+   */
+  @property({ attribute: 'brand-name' })
+  brandName?: string;
+
+  /**
+   * The g11n collator to use for sorting contry names.
+   */
+  @property({ attribute: false })
+  collatorCountryName = new Intl.Collator();
+
+  /**
+   * The throttle timeout to run query upon user input.
+   */
+  @property({ type: Number })
+  inputTimeout?: number;
+
+  /**
+   * The footer links.
+   */
+  @property({ attribute: false })
+  footerLinks: BasicLinkSet[] = [];
+
+  /**
+   * The language to show in the UI.
+   */
+  @property({ attribute: 'lang-display' })
+  langDisplay?: string;
+
+  /**
+   * The language used for query.
+   */
+  @property()
+  language?: string;
+
+  /**
+   * The legal nav links.
+   */
+  @property({ attribute: false })
+  legalLinks: BasicLink[] = [];
+
+  /**
+   * The locale list.
+   */
+  @property({ attribute: false })
+  localeList?: LocaleModalLocaleList;
+
+  /**
+   * The nonce used for logging in.
+   */
+  @property({ attribute: 'login-nonce' })
+  loginNonce?: string;
+
+  /**
+   * The link target to the masthead logo.
+   */
+  @property({ attribute: 'logo-href' })
+  logoHref?: string;
```

Comment:
I know this is probably a Masthead attribute, but not sure if we want to expose this as something configurable for authors, I think this should always go to www.ibm.com unless I'm mistaken? @wonilsuhibm @larahanlon2 @oliviaflory 

cc: @kennylam @annawen1 @andysherman2121 @RaphaelAmadeu

Reply:
👍 I would agree with @jeffchew, IBM logo should always to go ibm.com

Reply:
OK - Updated.